src/IndexerInterface.php

```diff
@@ -52,6 +58,7 @@ interface IndexerInterface
         TaoOntology::CLASS_URI_SUBJECT => self::TEST_TAKERS_INDEX,
         TaoOntology::CLASS_URI_TEST => self::TESTS_INDEX,
         self::MEDIA_CLASS_URI => self::ASSETS_INDEX,
+        'property-list' => 'property-list'
```

Comment:
Why we have this only hardcoded string at the array with a constant keys?

Reply:
This is very good question. **This string is representing new index** that I have to add here in order to be accepted by Indexer. This seems like huge restriction in library that I would like to highlight in order to fix it ASAP.